src/media.h

```diff
@@ -660,26 +678,26 @@ namespace OCICallbacks
 	  // show error result
 	  if ( !noFilelist_R.empty() ) //warning
 	  {
-	      cout << "Checking for file conflicts requires not installed packages to be downloaded in advance "
-	              "in order to access their file lists." << endl;
+	      qDebug() << "Checking for file conflicts requires not installed packages to be downloaded in advance "
+	              "in order to access their file lists." ;
 	      // print them out
-	      cout << "The following packages had to be excluded from file conflicts check because they are not yet downloaded" << endl;
+	      qDebug() << "The following packages had to be excluded from file conflicts check because they are not yet downloaded" ;
 	      for_( it, noFilelist_R.begin(), noFilelist_R.end() )
-		  cout << ( *it ) << endl;;
+		  ;//qDebug() << ( *it );
 	  }
 	  
 	  if ( !conflicts_R.empty() )
 	  {
 	      // use i18np() here
-	      cout << "Detected " << conflicts_R.size() << "file conflict(s)" << endl;
+	      qDebug() << "Detected " << conflicts_R.size() << "file conflict(s)" ;
 	      //print them out
 	      for_( it, conflicts_R.begin(), conflicts_R.end() )
```

Comment:
Instead of commenting the cout/qDebug statement, why noy commenting the whole for_ that now does nothing? (but waste cpu iterating to do nothing). The same some lines above